lib/Service/NextcloudVersionService.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Calendar\Service;
+
+use OCP\ServerVersion;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\ContainerInterface;
+
+class NextcloudVersionService {
+	private ?ServerVersion $serverVersion;
```

Comment:
```suggestion
	private ?ServerVersion $serverVersion = null;
```
Message: Typed property OCA\Calendar\Service\NextcloudVersionService::$serverVersion must not be accessed before initialization in file '/var/www/html/apps/calendar/lib/Service/NextcloudVersionService.php' line 25